lib/wit/deps/fastly/compute.wit

```diff
@@ -323,7 +323,7 @@ interface http-req {
 
   downstream-tls-ja4: func(max-len: u64) -> result<list<u8>, error>;
 
-  downstream-compliance-region: func(max-len: u64) -> result<list<u8>, error>;
+  downstream-compliance-region: func() -> result<list<u8>, error>;
```

Comment:
Could you add `max-len` back in? It's to help with the adapter, as it's ensuring that the returned vector that will be placed in the guest-provided buffer will be large enough.

Reply:
Done in 656a5a9.